.github/workflows/test.yml

```diff
@@ -0,0 +1,86 @@
+name: singularity-hpc-test-registry
+
+on:
+  pull_request: []
+    types: [labeled]
+    branches:
+      - recipe/*
+
+jobs:
+  test-registry:
+    if: (github.event.label.name == 'container-recipe') || startsWith(github.head_ref, 'recipe')
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get install lua5.2 liblua5.2-dev lua-filesystem lua-posix tcl tcl-dev
```

Comment:
you probably also want luajson and lua-term (but maybe they are already pulled in as deps).